### ISSUE TITLE
link tag incorrectly causes error for not having label

### ISSUE THREAD

**Issue by jnmorse** [jnmorse/eslint-config-jnmorse#16]
The "jsx-a11y/control-has-associated-label" rule falsely flags the link tag when used with react-helmet or next/head. This tag should be added to ignoreElements part of the rule.

**Comment by jnmorse** [comment]
4cba587 fixes this
